src/Pos/Block/Logic/Creation.hs

```diff
@@ -217,7 +217,9 @@ createMainBlockFinish slotId pske prevHeader = do
     (block, undo, pModifier) <-
         verifyCreatedBlock uncheckedBlock (pure unchecked) fallbackCreateBlock
     logDebug "Created main block/undos, applying"
-    lift $ block <$ applyBlocksUnsafe (one (Right block, undo)) (Just pModifier)
+    lift $ applyBlocksUnsafe (one (Right block, undo)) (Just pModifier)
+    logDebug "Applied successfully"
```

Comment:
I am afraid this message will look confusing for some people. There are many things which can be applied and we don't have good logger names nowadays, so one will have to search it in code. Maybe it's worth adding that we applied **a block ** successfully? Or maybe even attach header hash.